sdcm/sla/libs/sla_utils.py

```diff
@@ -375,3 +386,45 @@ def clean_auth(entities_list_of_dict):
                         auth.drop()
                     except Exception as error:  # pylint: disable=broad-except
                         LOGGER.error("Failed to drop '%s'. Error: %s", auth.name, error)
+
+    @staticmethod
+    def scylla_shares_per_scheduler_group(node, scheduler_group_name: str = None) -> dict:
```

Comment:
method name should tell the action (e.g. get_scylla_shares_per...).

Reply:
done